### DISCUSSION TITLE
What's the best way to access objects and change them?

### DISCUSSION THREAD

**Discussion by heberleh** [lineupjs/lineupjs#569] [Q&A]
I would like to search for Options, Columns, and so on and change them, however the attributes, such as `builder.columns`, are private, so I should not do that directly.

By replicating the code from the `demo/frozen.html`
```js
const b = LineUpJS.builder(arr);
b.options.flags.disableFrozenColumns = false;
b.deriveColumns();
b.column(LineUpJS.buildStringColumn('s').frozen());
b.build(document.body);
```

I get the message:

`Property 'options' is protected and only accessible within class 'LineUpBuilder' and its subclasses.`

Simply setting a column to `.frozen()` is not freezing it to the left side of LineUp; I guess this hard change in options is necessary.

Now, to combine this question with the 'how to find things to change':

- **How can I freeze the `aggregate` column and change its width?** (find it and change it)

Should I ignore type-warnings?

Many thanks!

**Comment by thinkh** [comment]
@sgratzl Can you please have a look and answer the question?
